oio/container/backup.py

```diff
@@ -781,6 +781,9 @@ def read(size):
             if inf.size % BLOCKSIZE:
                 read(BLOCKSIZE - inf.size % BLOCKSIZE)
 
+        if req_size != r['consumed']:
+            raise UnprocessableEntity()
```

Comment:
While this detects the error and fails, it would probably be better to specifiy in the returned UnprocessableEntity why it failed. For example a simple `Consumed size and expected size differ` would do the trick. _(Too many unprocessable entities are returned in the API, and it can be confusing to debug)_

Reply:
Indeed, I've added a small sentence for both empty UnprocessableEntity